infra/bridge/src/models/telegram-chat-log.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+
+module.exports = (sequelize, DataTypes) => {
+  const TelegramChatLog = sequelize.define(
+    'TelegramChatLog',
+    {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: DataTypes.INTEGER
+      },
+      message: {
+        type: DataTypes.STRING
+      },
+      userId: {
+        type: DataTypes.STRING
+      },
+      username: {
+        type: DataTypes.STRING
+      },
+      rawPayload: {
+        type: DataTypes.JSONB
+      },
+      created_at: {
+        type: DataTypes.DATE,
+        allowNull: false,
+        defaultValue: Sequelize.fn('now')
+      }
+    },
+    {
+      tableName: 'telegram_chat_logs'
+    }
+  )
+  TelegramChatLog.associate = function() {
```

Comment:
Just curious, what is this for?

Reply:
This is where we define model associations, if any. Since this model doesn't have any association, as of now, I have left that empty.

Reply:
aha I see thanks